test/sys/test_epoll.rs

```diff
@@ -1,24 +1,39 @@
+use std::os::unix::io::AsRawFd;
+
+use nix::{Error, Errno};
 use nix::sys::epoll::{EpollCreateFlags, EpollOp, EpollEvent};
 use nix::sys::epoll::{EPOLLIN, EPOLLERR};
 use nix::sys::epoll::{epoll_create1, epoll_ctl};
-use nix::{Error, Errno};
+use nix::sys::signal::SigSet;
+use nix::sys::signalfd::SignalFd;
+use nix::unistd::close;
 
 #[test]
 pub fn test_epoll_errno() {
+    // Create a new signalfd to use for testing. This is better than using stdin or other such fd.
+    let sfd = SignalFd::new(&SigSet::empty()).unwrap();
```

Comment:
is a `SignalFd` really the simplest kind of file descriptor that works with `epoll`?  What about plain old files?